cmd/kfulint/main.go

```diff
@@ -123,11 +113,20 @@ func (l *linter) LoadProgram() {
 }
 
 func (l *linter) InitCheckers() {
-	for _, name := range lint.AvailableCheckers() {
-		// Nil enabledSet means "all checkers are enabled".
-		if l.enabledSet == nil || l.enabledSet[name] {
-			l.checkers = append(l.checkers, checker{name, lint.NewChecker(name, l.ctx)})
+	if l.enabledCheckers == nil {
+		for _, name := range lint.AvailableCheckers() {
+			c, _ := lint.NewChecker(name, l.ctx)
+			l.checkers = append(l.checkers, checker{name, c})
+		}
+	}
+
+	for _, s := range l.enabledCheckers {
+		name := strings.TrimSpace(s)
+		c, ok := lint.NewChecker(strings.TrimSpace(name), l.ctx)
```

Comment:
I think double trimming is a mistake.

Reply:
It is first time where it trimmed

Reply:
`name := strings.TrimSpace(s)`
Then `lint.NewChecker(strings.TrimSpace(name), l.ctx)`.

Reply:
`name` is trimmed twice.

Reply:
oh, i see. sry